olive/systems/azureml/aml_pass_runner.py

```diff
@@ -103,12 +102,24 @@ def main(raw_args=None):
     # this is to handle passes like OrtPerfTuning that use the same model file as input
     model_json["same_model_path_as_input"] = False
     if model_json["config"]["model_path"] is not None:
-        model_path = str(Path(model_json["config"]["model_path"]).resolve())
-        if model_path == input_model_path:
+        # create a resource path from the model path
+        model_resource_path = ResourcePath.create_resource_path(model_json["config"]["model_path"])
```

Comment:
do we need assert the model_resource_path is local resource?

Reply:
It gets checked indirectly when the pass, model or evaluator calls `model.model_path` https://github.com/microsoft/Olive/blob/jambayk/model-source/olive/model.py#L67

Reply:
Done